src/vs/workbench/contrib/terminalContrib/chat/browser/terminal.chat.contribution.ts

```diff
@@ -11,6 +11,9 @@ import { TerminalChatController } from 'vs/workbench/contrib/terminalContrib/cha
 
 registerTerminalContribution(TerminalChatController.ID, TerminalChatController, false);
 
+registerWorkbenchContribution2(TerminalChatEnabler.Id, TerminalChatEnabler, WorkbenchPhase.AfterRestored);
```

Comment:
This is a workbench contribution so it should be in the "Contributions" section below